scripts/us_bjs/nps/preprocess_data.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
+import argparse
+
+STAT_VAR_NAMES = ["Count_Jurisdiction_WhiteAlone_Female", "Count_Jurisdiction_WhiteAlone_Male", "Count_Jurisdiction_WhiteAlone","Count_Jurisdiction_BlackOrAfricanAmericanAlone_Female",
```

Comment:
All these StatVars should include Incarcerated or something indicating this is a count of imprisoned people.